ui/src/playlist/PlaylistActions.js

```diff
@@ -95,10 +124,14 @@ const PlaylistActions = ({ className, ids, data, record, ...rest }) => {
   return (
     <TopToolbar className={className} {...sanitizeListRestProps(rest)}>
       <Button
-        onClick={handlePlay}
-        label={translate('resources.album.actions.playAll')}
+        onClick={playing ? () => dispatch(pausePlayer()) : handlePlay}
```

Comment:
> Yes, this is much better. The reason I did this here is that this is the `PlayArrowIcon` and `PauseIcon` from Material UI instead of using PlayButton from `/ui/src/common`. I hope this doesn't break using /common/PlayButton in other places. But, I understood the change. Will do the needful.

Will do the same here